examples/gno.land/p/demo/gnorkle/gnorkle/storage.gno

```diff
@@ -0,0 +1,11 @@
+package gnorkle
+
+import "gno.land/p/demo/gnorkle/feed"
+
+// Storage defines how published feed values should be read
+// and written.
+type Storage interface {
+	Put(value string)
+	GetLatest() feed.Value
+	GetHistory() []feed.Value
```

Comment:
I can see your point. I haven't made a decision yet as to how this data should be exposed, but an iterator makes sense.  I guess that if a realm would like to grant someone access to all the historical data for a feed with one function call, then the realm could provide such a function for iterating over the data and returning the full result. So yes, I can change this so that it returns an iterator.